module/zfs/txg.c

```diff
@@ -784,22 +776,23 @@ txg_wait_open(dsl_pool_t *dp, uint64_t txg, boolean_t should_quiesce)
 }
 
 /*
- * If there isn't a txg syncing or in the pipeline, push another txg through
- * the pipeline by quiescing the open txg.
+ * If there isn't a txg quiescing in the pipeline, push the txg
+ * through the pipeline by quiescing the open txg.
+ * It is fine there is a txg still syncing.
```

Comment:
Can you explain here that the caller should pass the txg number of the transaction that should be closed and synced.

Reply:
Instruction will be added in next commit.